### DISCUSSION TITLE
Save to use --start-domain option for running VM?

### DISCUSSION THREAD

**Discussion by slu-010101** [abbbi/virtnbdbackup#165] [Q&A / Support] [answered]
Hi,

it's not clear to me i can use this option on a running VM, man virtnbdbackup say:
```
       -S, --start-domain
              Start virtual machine if it is offline. (default: False)
```

But what happen if the VM already running, will it destroyed after the backup?

https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backing-up-offline-virtual-domains

Thanks.

**Comment by abbbi** [comment, marked as answer]
the option does not have any effect if domain is already running

no operation will be carried out, nothing will be destroyed.

**Comment by slu-010101** [comment]
Ok so it's safe to use this option as default in my setup, thanks.

Can you add this information here:
https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backing-up-offline-virtual-domains

**Reply by abbbi** [reply]
https://github.com/abbbi/virtnbdbackup/commit/12c582e14cb844167bcce5dd00bf99c418b58c3b
